modules/ROOT/pages/release-notes/release-notes57.adoc

```diff
@@ -151,23 +154,23 @@ For information on the Spell Checker Pro API, see: link:{baseurl}/plugins/premiu
 * `mceSpellcheckDialog`
 * `mceSpellcheckDialogClose`
 
-For information on the Spell Checker Pro Commands, see: link:{baseurl}/plugins/premium/tinymcespellchecker/#commands[Spell Checker Pro - Commands].
+For information on the Spell Checker Pro Commands, see: xref:plugins/premium/tinymcespellchecker.adoc#commands[Spell Checker Pro - Commands].
 
 ===== New Spell Checker Pro Event
 
 *Spell Checker Pro* introduces a new `SpellcheckerLanguageChanged` event which is fired when the active language is changed.
 
-For information on the Spell Checker Pro Events, see: link:{baseurl}/plugins/premium/tinymcespellchecker/#events[Spell Checker Pro - `Events`].
+For information on the Spell Checker Pro Events, see: xref:plugins/premium/tinymcespellchecker.adoc#events[Spell Checker Pro - `Events`].
```

Comment:
This xref doesn't resolve as the `events` id doesn't exist in `tinymcepspellchecker.adoc`. It needs to be added to the relevant heading. e.g:
```
[[events]]
== Events
```

Reply:
done